mappings/net/minecraft/client/color/block/BlockColorProvider.mapping

```diff
@@ -1,5 +1,7 @@
 CLASS net/minecraft/class_322 net/minecraft/client/color/block/BlockColorProvider
 	METHOD getColor (Lnet/minecraft/class_2680;Lnet/minecraft/class_1920;Lnet/minecraft/class_2338;I)I
+		COMMENT {@return the color of the block state for the specified tint index,
+		COMMENT or -1 if not tinted}
```

Comment:
hmm, is -1 just fully white color, so it's effectively not tinted?

Reply:
Technically, yes. However, the actual constant is still probably -1, as using it defines the alpha value which is usually left out of the color provider return values. (The actual alpha is always ignored and set to 0xFF)

Reply:
I guess they must have this as a constant somewhere... Hope we can unpick it at some point in the future

Reply:
It's the private constant that I mapped in BlockColors and ItemColors. Couldn't get it to unpick, though - maybe some lambda issues?

Reply:
Hmm yes, lambda methods don't explicitly override the single abstract method in the interface, and detection does need to check each bootstrap method (or indy callsite) manually, and I doubt if unpick even has that check